renovate.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema":"https://docs.renovatebot.com/renovate-schema.json",
+  "baseBranches":[
+     "main",
+     "release-v0.20",
```

Comment:
You can specify the branch names with a wild card.
For example:
  "baseBranches": ["/^rhel-9.\\d$|^rhel-10.\\d|^rhel-10.0-beta$/"]

Reply:
We want to update the security dependencies only in some branches, not in all release-v... branches. Thats why I put there the names instead of regex.